components/renku_data_services/notebooks/api/amalthea_patches/git_proxy.py

```diff
@@ -67,9 +70,9 @@ async def main_container(
     container = client.V1Container(
         image=config.sessions.git_proxy.image,
         security_context={
-            "fsGroup": 100,
-            "runAsGroup": 1000,
-            "runAsUser": 1000,
+            "fsGroup": fs_group,
```

Comment:
`fsGroup` is part of [PodSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#podsecuritycontext-v1-core), not part of [SecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core).

That means it applies to the pod spec but not to the containers specs.

Reply:
Removed it